AdjustedSurvivalCurves/adjcurves_plain.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+title: "Adjusted survival curves"
+author: André Moser, CTU Bern, University of Bern
+output: github_document
+bibliography: adjcurves_plain.bib
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Background
+
+Unadjusted Kaplan-Meier curves from observational data might be biased because of confounding (@COLE200445). This vignette introduces the concept of inverse probability (IP) weighted survival curves.
+
+# Example
+
+We consider a study population of 10,000 patients who received a medication. 60% of the patients were women. In men, a specific enzym was measured with a probability of 30%, whereas in women have a 5 times higher chance of having the enzym. Those with the enzym were more likely to receive the medication (with a probability of 75%), compared to those without the enyzm (probability of 50%).
+
+- Research question: What is the effect of the medication on death?
+- Study design: Cohort study
+- Outcome of interest: Time to death or end of follow-up
+- Predictor of interest: Medication
+- Confounders: Sex, enzym
+
+The used variables from the data are:
+
+| Variable | Definition | Coding |
+| --- | --- | --- |
+| female | Sex | 1=Women, 0=Men |
+| enzym | Measured enzym | 1=Present, 0=Not present |
+| medi | Medication | 1=Received, 0=Not received |
+| death | Death | 1=Death, 0=Alive |
+| fup | Follow-up time | Non-negative number |
+
+## Knowledge from the crystal ball
+
+Because we simulated the data, we know that medication has no effect on mortality.
+
+# Analysis strategy
+
+IP weighting constructs weights which are equal to the probability of an individual's treatment (here: receiving the medication) given observed covariates (here: sex and enzym) and creates pseudopopulations in which observed covariates are independent of treatment (i.e. no confounding) (@hernan_book).
+
+```{r, echo=T, message=F, warning=F}
+# Required packages
+library(tidyverse)
+library(survey)
+library(survival)
+library(gtsummary)
+library(survminer)
+```
+
+```{r, echo=F, message=F, warning=F, results='asis'}
+set.seed(1)
+n <- 10000
```

Comment:
While it might be very useful for illustration purposes, I find 10'000 patients a bit too far away from a realistic practical setting.